test/hmactest.c

```diff
@@ -214,6 +214,29 @@ static int test_hmac_single_shot(void)
     return 1;
 }
 
+/* https://github.com/openssl/openssl/issues/13210 */
+static int test_hmac_benmocha(void)
+{
+    HMAC_CTX *ctx = NULL;
+    unsigned char buf[EVP_MAX_MD_SIZE];
+    unsigned int len;
+    int ret = 0;
+
+    /* HMAC_Update() after HMAC_Final() must return an error. */
+    ctx = HMAC_CTX_new();
```

Comment:
This needs to be TEST_ptr_null since HMAC_Init_ex() does not check for ctx being NULL.

Reply:
No, this should be wrapped in `TEST_ptr()` so a meaningful error message is produced on test failure:
```c
    if (!TEST_ptr(ctx = HMAC_CTX_new()))
        goto err;
```